### DISCUSSION TITLE
Significant Improvements 💪

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#387] [Announcements]
In the process of creating the EpicWeb.dev workshops, I've made a number of improvements to the stack which have now been applied.

Decision documents have been made for the biggest things:

- https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/026-path-aliases.md
- https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/027-toasts.md
- https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/028-permissions-rbac.md
- https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/029-remix-auth.md
- https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/030-github-auth.md

If you're trying to keep your fork of the Epic Stack up-to-date, I'm afraid to say you will have a very difficult time with this update. As usual, I don't recommend doing this. Be content with the stack as it was and manually apply only the changes that are necessary for the improvements you would like.

**Comment by cr101** [comment]
All great improvements. Thanks Kent for your hard work.
